src/TRE-API/Controllers/TRINOCredentialsController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using BL.Models.APISimpleTypeReturns;
+using BL.Models;
+using BL.Models.Settings;
+using Microsoft.AspNetCore.Authorization;
+using BL.Services;
+using Microsoft.AspNetCore.Mvc;
+using TRE_API.Repositories.DbContexts;
+using TRE_API.Services;
+
+namespace TRE_API.Controllers
+{
+    [Route("api/tre-credentials/[controller]")]
```

Comment:
```suggestion
    [Route("api/tre-credentials/trino")]
```

Better matches what is done in #927